app/api/v1/endpoints/webhooks.py

```
@@ -0,0 +1,185 @@
+    data = WebhookSubscriptionUpdate(
+        target_url=body.target_url,
+        event_types=body.event_types,
+        subject_types=body.subject_types,
+        secret=body.secret,
+        active=body.active,
+    )
+    try:
+        updated = await repo.update(tenant_id, subscription_id, data)
+        return _to_response(updated)
+    except ResourceNotFoundException:
+        raise HTTPException(status_code=404, detail="Webhook subscription not found")
```

Ruff (0.15.5)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)